### DISCUSSION TITLE
Stress test for undervolt

### DISCUSSION THREAD

**Discussion by Inaiso** [seerge/g-helper#685] [Q&A]
I wanted to know what free programs are good to stress the system after undervolt to check for stability.

**Comment by seerge** [comment]
@Inaiso i heard people use Prime95 to stress test

**Comment by Wolper321** [comment]
I used CinebenchR23 single thread test to find my -10 stable point for CPU.
iGPU at the other hand is more capricious and it's not crashing the system but rather than amd drivers. Furmark doesn't show the full picture you need to play the games with full load.
